tests/datasets/test_events_processor.py

```diff
@@ -140,8 +141,8 @@ def test_v2_start_delete_groups(self):
         processor = (
             enforce_table_writer(self.dataset).get_stream_loader().get_processor()
         )
-        assert processor.process_message(message) == ProcessedMessage(
-            action=ProcessorAction.REPLACE, data=[(str(project_id), message)],
+        assert processor.process_message(message) == ReplacementBatch(
+            str(project_id), [message]
         )
```

Comment:
It'd probably make sense to parametrize these tests at some point but didn't want to increase the +/- of this change.